lib/unit-ruby/util/resource_operations.rb

```diff
@@ -77,5 +77,25 @@ def save(attributes)
         update_resource_from_json_api(updated_resource)
       end
     end
+
+    # Replace the entire resource, instead of only patching dirty attributes
+    module Replace
```

Comment:
Since a PUT is intended to replace a given resource, while PATCH is a partial update (e.g. Save), I named this resource operation Replace.